src/posix/platform/system.cpp

```diff
@@ -303,14 +329,29 @@ void platformDeinit(void)
     return;
 }
 
-void otSysDeinit(void)
+void platformDeinitNcpMode(void)
 {
-    OT_ASSERT(gInstance != nullptr);
+#if OPENTHREAD_POSIX_VIRTUAL_TIME
+    virtualTimeDeinit();
```

Comment:
Do we need this call to `virtualTimeDeinit()` here?

We don't seem to init it from `platformInitNcpMode()`? Does it get initialized somewhere else?

Reply:
Good catch! Have removed it.